File: app/dbmodels/InstalledSoftwareVersion.php
<?php

$table = array(
    'fields'        => array(),
    'oldfields'     => array(),
    'renamefields'  => array(),
    'indexes'       => array(),
    'oldindexes'    => array(),
    'relationships' => array(),
    'model'         => 'InstalledSoftwareVersion',
    'menu'          => ''
);

$table['fields'] = array(
    'id'                  => DBModels::type('increments', 
                                            array('visible' => false)),
    'asset_id'            => DBModels::type('integer'),
    'software_version_id' => DBModels::type('integer'),
    'is_deleted'          => DBModels::type('boolean'),
    'is_dynamic'          => DBModels::type('boolean'),
    'is_deleted_asset'    => DBModels::type('boolean'),
    'is_template_asset'   => DBModels::type('boolean'),
);


include "_commonentitylink.php";
include "_commonrecursiveentitylink.php";

File: fusionglpi/database/installed_software_version.py
from fusionglpi import db

class InstalledSoftwareVersion(db.Model):
    id = db.Column(db.Integer, primary_key=True)
    entity_id = db.Column(db.Integer, db.ForeignKey('entity.id'))
    is_recursive = db.Column(db.Boolean, nullable=False, server_default="0")
    asset_id = db.Column(db.Integer, db.ForeignKey('asset.id'))
    software_version_id = db.Column(db.Integer, db.ForeignKey('software_version.id'))
    is_deleted = db.Column(db.Boolean, nullable=False, server_default="0")
    is_dynamic = db.Column(db.Boolean, nullable=False, server_default="0")
    is_deleted_asset = db.Column(db.Boolean, nullable=False, server_default="0")
    is_template_asset = db.Column(db.Boolean, nullable=False, server_default="0")
